src/AppBundle/Repository/AutoEditsRepository.php

```diff
@@ -278,19 +307,14 @@ public function getAutomatedEdits(
      *                      ],
      *                  ]
      */
-    public function getToolCounts(
-        Project $project,
-        User $user,
-        $namespace = 'all',
-        $start = '',
-        $end = ''
-    ): array {
+    public function getToolCounts(Project $project, User $user, $namespace = 'all', $start = '', $end = ''): array
```

Comment:
personally, I would prefer to have such long method declarations be split onto multiple lines, but either way suggest doing separately from this patch (as far as I can tell this is a whitespace only change, so it should have no impact on the result and just makes the diff a bit harder to review)